webpack.dev.js

```diff
@@ -13,3 +13,9 @@ module.exports = merge(common, {
         host: '0.0.0.0',
     },
 })
+
+config.module.rules[3].use[1].options.modules = {
+    localIdentName: '[path][name]__[local]'
```

Comment:
I took this from the documentation here: https://webpack.js.org/loaders/css-loader/#localidentname
Not really sure we absolutely need path and name and we could also use local+hash, but also no real reason not to use the recommended format?